### PR TITLE
cli: create user with profile

### PR DESCRIPTION
**Pull request by dfdan**
:heart: Thank you for your contribution!

### Description

Adds -p/--profile option to `invenio users create` to supply a profile at time of user creation. This enables users to be created on the command line with their full name (and/or other profile items set) configured. 

eg `invenio users create -a -c -p '{"full_name" : "Test User"}'  test@example.com`

(Motivation: a user created above is created with enough details set to make the searchable, in the /api/users endpoint, without the need for the user to modify their profile (nb: also requires default profile/email visibility to be set to public in invenio.cfg))

Discussed with @slint.

### Checklist

Ticks in all boxes and 🟢 on all GitHub actions status checks are required to merge:

- [x] I'm aware of the [code of conduct](https://inveniordm.docs.cern.ch/contribute/code-of-conduct/).
- [x] I've created [logical separate commits](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#commits) and followed the [commit message format](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#commit-message).
- [x] I've added relevant test cases.
- [x] I've added relevant documentation.
- [x] I've marked [translation strings](https://inveniordm.docs.cern.ch/develop/best-practices/i18n/) (for relevant code).
- [x] I've followed the [CSS/JS](https://inveniordm.docs.cern.ch/develop/best-practices/css-js/) and [React](https://inveniordm.docs.cern.ch/develop/best-practices/react/) guidelines (for relevant code).
- [x] I've followed the [web accessibility](https://inveniordm.docs.cern.ch/develop/best-practices/accessibility/) guidelines (for relevant code).
- [x] I've followed the [user interface](https://inveniordm.docs.cern.ch/develop/best-practices/ui/) guidelines (for relevant code).
- [x] I've identified the [copyright holder(s)](https://inveniordm.docs.cern.ch/contribute/copyright-policy/) and updated copyright headers for touched files (>15 lines contributions).
- [x] I've NOT included third-party code (copy/pasted source code or new dependencies).

**Third-party code**

If you've added [third-party code (copy/pasted or new dependencies)](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#third-party-codedependencies), please reach out to an [architect](https://github.com/orgs/inveniosoftware/teams/architects/members).

**Reminder**

By using GitHub, you have already agreed to the [GitHub’s Terms of Service](https://help.github.com/articles/github-terms-of-service/#6-contributions-under-repository-license) including that:

1. You license your contribution under the same terms as the current repository’s license.
2. You agree that you have the right to license your contribution under the current repository’s license.

### CODE REVIEW

File: invenio_accounts/cli.py [hunk 1]
```diff
@@ -61,6 +63,8 @@ def users_create(email, password, active, confirm):
         kwargs["active"] = active
         if confirm:
             kwargs["confirmed_at"] = datetime.utcnow()
+        if profile:
+            kwargs["user_profile"] = json.loads(profile)
```
Comment (ntarocco) [inline invenio_accounts/cli.py:67]: Unfortunately, `invenio-accounts` does not depend on `invenio-userprofiles`, but the other way around. So, it should not be possible (ideally) to do this, in particular if the `user_profile` schema will change.
Was this discussed?

Comment (slint) [reply invenio_accounts/cli.py:67]: [`user_profile` is not part of the accounts model](https://github.com/inveniosoftware/invenio-accounts/blob/e2759b5a56181df27ed750f1e336fad84e0752b4/invenio_accounts/models.py#L119-L125). So as long as the passed JSON conforms to the configurable schema we're validating the input properly.

Comment (ntarocco) [reply invenio_accounts/cli.py:67]: > [`user_profile` is part of the accounts model](https://github.com/inveniosoftware/invenio-accounts/blob/e2759b5a56181df27ed750f1e336fad84e0752b4/invenio_accounts/models.py#L119-L125). So as long as the passed JSON conforms to the configurable schema we're validating the input properly.

Oh, thanks for reminding me that, I was convinced it was not, didn't even check :) Sorry about that!

File: tests/test_cli.py [hunk 2]
```diff
@@ -33,6 +33,23 @@ def test_cli_createuser(app):
...
+    # Create use with profile that will not validate
+    result = runner.invoke(
+        users_create,
+        [
+            "test@example.com",
+            "--profile",
+            '{"full_name" : "test_user", "extra_data" : "not_allowed"}',
+        ],
+    )
+    assert result.exit_code != 0
```
Comment (slint) [inline tests/test_cli.py:51]: minor: I think we also need a test-case for the happy path, i.e. passing a correct `--profile` value that then gets set on the created user.

Comment (chokribr) [reply tests/test_cli.py:51]: coordination: @slint please take a look if  the test pass please approve?

**Pull request merged**